proxygen_cli/cli/command_spec.py

```diff
@@ -13,15 +14,26 @@
 
 @click.group()
 @click.option(
-    "--api", default=SETTINGS.api, help="Override the default API", show_default=True
+    "--api",
+    default=SETTINGS.api,
+    help="Override the default API",
+    show_default=True
+)
+@click.option(
```

Comment:
Annoyingly, there's no way to add the `--uat` option to the root command and have it appear in the docs for both:
- `proxygen spec --help`
- `proxygen spec publish --help`
The only way around this is to add the `--uat` option to each sub-command (publish, get, delete) and then also to the root function.. and ignore the value.